tests/integration/test_data/bundler_everything_present_except_gemspec/.build-config.yaml

```diff
@@ -9,3 +9,5 @@ project_files:
     BUNDLE_DEPLOYMENT: "true"
     BUNDLE_NO_PRUNE: "true"
     BUNDLE_VERSION: "system"
+    BUNDLE_DISABLE_LOCAL_BRANCH_CHECK: "true"
+    BUNDLE_LOCAL.JSON___SCHEMA: "${output_dir}/deps/bundler/json-schema-26487618a684"
```

Comment:
Uhm, would it be possible for us to make the ref segment of the file name a placeholder too? I guess not, anyway, not a blocker it's just that whenever the test repo is updated it'll break the ITs again requiring and explicit update of the ref. Also, the convenience test data generator would not be able to cope with it. It is what it is....